### DISCUSSION TITLE
Change Game Sounds?

### DISCUSSION THREAD

**Discussion by Baumhausbanane123e** [Maerciez/TheMillionaireGame#85] [Q&A] [answered]
Hello,
In the Game folder there are a bunch of sounds (lots of them unused) and in the game settings tewo options for sounds with each having a "description" for the sound and the file name in a field tha is non changable. But how do you actually change the sounds?
Besides replacing the files obviously.

**Comment by Macronair** [comment, marked as answer]
The textboxes in the options screen are indeed not directly changeable. But when you double click one of the textboxes in both Sounds tabs, a window will appear where you can select another sound file.
(Don't forget to save the settings after changing this).

MP3, WAV and FLAC files are working the best in the application in case you have your own sounds.
